pages/api/review.ts

```diff
@@ -3,49 +3,57 @@ import mongoose from "mongoose"
 
 
-export default async function handler(req: NextApiRequest, res: NextApiResponse){
+export default async function handler(req: NextApiRequest, res: NextApiResponse,) {
 
-    
-    try{
+    //publically accessible API endpoints
 
-        
-        if(req.method==="POST"){        //so only posts request make it through
-            if(!req.body.review || req.body.subject==='-' || req.body.level==='-' || req.body.review.length<20) return res.status(400).send("Missing post body");
+    try {
+        if (req.method === "POST") {        //so only posts request make it through
+            if (!req.body.review || req.body.subject === '-' || req.body.level === '-' || req.body.review.length < 20) return res.status(400).send("Missing post body");
             console.log(req.body.name)
             console.log(process.env.MONGODB_URL)
             await mongoose.connect(process.env.MONGODB_URL as string);   //establish a connection with the database
             await ReviewModel.create({
                 review:
-                    {original:req.body.review}, //create new post item using the body datafield of the request
-                subject: 
-                    {original: req.body.subject},
-                name: 
-                    {original: req.body.name},
-                level: 
-                    {original: req.body.level},
+                    { original: req.body.review }, //create new post item using the body datafield of the request
+                subject:
+                    { original: req.body.subject },
+                name:
+                    { original: req.body.name },
+                level:
+                    { original: req.body.level },
                 displayed: false,            //by default it is not displayed in the feedback page yet
             })
-            return res.status(200).send("Success") 
-        }else if (req.method==='GET'){
+            return res.status(200).send("Success")
+        } else if (req.method === 'GET') {
 
             await mongoose.connect(process.env.MONGODB_URL as string);
             const reviews = await ReviewModel.find();
 
-            return res.status(200).json({reviewsArray: reviews})
 
-        }else if (req.method==='PUT'){        //update the review information
-            await mongoose.connect(process.env.MONGODB_URL as string);
-            await ReviewModel.findOneAndUpdate({_id: req.body._id} ,  //filter condition then the desired update
-                {displayed: req.body.displayed, "review.current": req.body.reviewLatestEdit, "subject.current": req.body.subjectLatestEdit,
-                "name.current": req.body.nameLatestEdit, "level.current": req.body.levelLatestEdit, date: req.body.date, examBoard: req.body.examBoard})       //review.current gets the nested field review:{current:}      
+            //set all stuff to 'original' even if edited, mabye should rename 'original' to not be misleading
```

Comment:
You could have `review`, `name`, `subject` etc. just be strings instead of objects - no need to have the same interface for `/api/review` and `/api/review-admin`.

Reply:
That makes sense, so I should also update it in the 'feedback' page too, to not have review.original etc?

Reply:
Yes, anywhere you use `/api/review` you'll need to update this.

Reply:
oh wait i think the red thing i moved to review-admin. But I can still get rid of review:{original: ...} and just do review: ...